src/shared/cloudformation/cloudformation.ts

```diff
@@ -502,9 +502,9 @@ export namespace CloudFormation {
      * Validates whether or not a property is a valid ref.
      * @param property Property to validate
      */
-    function isRef(property: string | number | object | undefined): boolean {
+    function isRef(property: unknown): boolean {
         return (
-            typeof property === 'object' && Object.keys(property).length === 1 && Object.keys(property).includes('Ref')
+            typeof property === 'object' && Object.keys(property as Record<string, unknown>).length === 1 && Object.keys(property as Record<string, unknown>).includes('Ref')
```

Comment:
Is there a reason to drop all of the `object` types? I especially don't understand it here: this should make the assertion that the `property` is already an object by this point, so `Object.keys` should be fine?

Reply:
This is a new ESLine rule. I mean we can disable it if you think it's best. `object` is basically equivalent to `any` as far as ESLint is concerned.

Reply:
in this case `as any` should work

Reply:
Ah that's the thing, ESLint wants you to not use `any` as much as possible. `Record<string, unknown>` is considered the preferred substitute for `object`

Reply:
yuck

Reply:
Adding non-null assertion made the compiler correctly infer `object` as a type (with no ESLint error)